src/plugins/lua/luaplugin.cpp

```diff
@@ -198,61 +258,67 @@ static bool includeTile(const Tile *tile)
     return false;
 }
 
-void LuaPlugin::writeTileset(LuaTableWriter &writer, const Tileset *tileset,
-                             unsigned firstGid)
+void LuaUtilWriter::writeTileset(LuaTableWriter &writer, const Tileset &tileset,
+                                    unsigned firstGid, bool standalone)
 {
-    writer.writeStartTable();
+    if (standalone) {
+      writer.writeStartTable();
+    } else {
+      writer.writeStartReturnTable();
+    }
 
-    writer.writeKeyAndValue("name", tileset->name());
-    writer.writeKeyAndValue("firstgid", firstGid);
+    writer.writeKeyAndValue("name", tileset.name());
+    if (standalone) {
+      writer.writeKeyAndValue("firstgid", firstGid);
+    }
```

Comment:
Coding style: in case of single-line body, please leave out the brackets (also above).

Reply:
Done